packages/functional-tests/pages/login.ts

```diff
@@ -14,6 +14,7 @@ export const selectors = {
   LINK_USE_RECOVERY_CODE: '#use-recovery-code-link',
   NUMBER_INPUT: 'input[type=number]',
   PASSWORD: '#password',
+  SHOW_PASSWORD: '#password ~ [for="show-password"]',
```

Comment:
Does this need to be added to `LoginPage.selectors` below?
Although I'm not 100% convinced the `readonly selectors` object from L43-76 is&hellip; used.

Reply:
removed the `LoginPage.selectors` overall.